### ISSUE TITLE
Output to stdout, and pipe, rather than overwrite

### ISSUE THREAD

**Issue by patarapolw** [RocktimSaikia/syzer#4]
```sh
npx syzer -o > package.json
```

- This makes it easy to edit programmatically with [jq](https://stedolan.github.io/jq/), or manually.
- `npx syzer`, without `-u`, is not easy for grepping
- [PNPM](https://pnpm.js.org/) does support `package.yaml` and `package.json5`

Also, piping in would also be beneficial, because of `jq` as well.

```sh
cat package.json | jq '.dependencies' | npx syzer
```

**Comment by rocktimsaikia** [comment]
@patarapolw Agreed! Initially, when I first created this tool, I did not have much knowledge on how things work in nodejs but still, I didn't like the existing tools that do this job so just tried to build this quick module that does the job for me personally and I still do use it. Now looking back at the project I see that the project can be improved in various ways. I tried to write code for every little thing when there are many utility modules that can be used here. 
I didn't think anyone would actually use it so was not planning to update it, now I see that it would be a good time to update the project.
In the meantime, PRs are welcome for this issue

**Comment by patarapolw** [comment]
[I implemented my thought (for another cli app) here.](https://github.com/patarapolw/jy/blob/master/cli.js) It will took a while to port to syzer.
